<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edge list</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>4</y>
     <width>401</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vl">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addEdge">
        <property name="text">
         <string>Add edge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteEdge">
        <property name="text">
         <string>Delete edge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveChanges">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="table">
      <property name="columnCount">
       <number>4</number>
      </property>
      <column>
       <property name="text">
        <string>Start point</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>End point</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Weigth</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Is directed</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
